packages/shared/src/common/retry.ts

```diff
@@ -1,64 +1,133 @@
-import { errorToString } from "./error";
+import { random } from "lodash";
+import { MetriportError } from "../error/metriport-error";
+import { errorToString } from "../error/shared";
 import { sleep } from "./sleep";
 
+export const defaultOptions: Required<ExecuteWithRetriesOptions> = {
+  initialDelay: 10,
+  maxDelay: Infinity,
+  backoffMultiplier: 2,
+  maxAttempts: 10,
```

Comment:
I was going with 5, but ended up following https://www.npmjs.com/package/exponential-backoff

The thing with a low number, is that we would prob need to increase the initial delay as well, otherwise we would quickly stop retrying - and if a service is down, in theory we want to try for a certain period of time to maximize the chances of a successful request.

Thoughts?